app/models/agents/gap_detector_agent.rb

```diff
@@ -8,6 +8,11 @@ class GapDetectorAgent < Agent
       The `value_path` value is a [JSONPath](http://goessner.net/articles/JsonPath/) to a value of interest. If either
       this value is empty, or no Events are received, during `window_duration_in_days`, an Event will be created with
       a payload of `message`.
+
+      The `stream_path` value is a [JSONPath](http://goessner.net/articles/JsonPath/) to a value in the event that will
```

Comment:
The main reason I asked about the change was that I did not understand how the Agent is supposed to work by only reading the description, and based on his first commend i guess @cantino felt the same.

I might have made the mistake to talk about the code too much, where I really meant to talk about the configuration and documentation of the agent. The agent needs a path (read data) to operate upon and does things differently based on a "mode" or "behavior", that is how it picture it as a user.

We probably need a tie breaker, otherwise this discussion will never come to a conclusion :)